pkg/kubernetes-mcp-server/cmd/root.go

```diff
@@ -47,9 +47,34 @@ kubernetes-mcp-server --port 8080
 
 # start a SSE server on port 8443 with a public HTTPS host of example.com
 kubernetes-mcp-server --port 8443 --sse-base-url https://example.com:8443
+
+# start a SSE server on port 8080 with multi-cluster tools disabled
+kubernetes-mcp-server --port 8080 --disable-multi-cluster
 `))
 )
 
+const (
+	flagVersion              = "version"
```

Comment:
these are nice, but I'd have perhaps done this in a separate PR before. Like a "pre factoring". 

But since the change is small, this is adding not too much noise. Just as a general statement